client/extensions/woocommerce/app/order/order-fulfillment/index.js

```diff
@@ -135,17 +141,44 @@ class OrderFulfillment extends Component {
 		}
 	};
 
+	isAddressValidForLabels( address ) {
+		const { labelCountriesData } = this.props;
+		const countryData = labelCountriesData[ address.country ];
+
+		//country not supported
+		if ( ! countryData ) {
+			return false;
+		}
+
+		//supported country doesn't have a states data
+		if ( ! countryData.states ) {
+			return true;
+		}
+
+		//check if the address state is supported
+		return Boolean( countryData.states[ address.state ] );
+	}
+
+	shouldShowLabels() {
+		const { labelsLoaded, labelsEnabled, order, storeAddress, hasLabelsPaymentMethod } = this.props;
+
+		if ( ! wcsEnabled || ! labelsLoaded ) {
```

Comment:
looks like `labelsLoaded` checks for truthyness of `wcsEnabled` so couldn't this just be `if ( ! labelsLoaded )`?